CONTRIBUTING.md

```diff
@@ -146,8 +146,7 @@ Create the conda development environment:
 ```bash
 # create the conda environment (assuming in base `clx` directory)
 
-conda env create --name clx_dev --file conda/environments/clx_dev_cuda11.0.yml
-
+mamba env update --name clx_dev --file conda/environments/clx_dev_cuda11.5.yml
```

Comment:
there are other spots in `CONTRIBUTING` and also `README` where we need update from `11.0` to `11.5`.

Reply:
also should be `mamba env create`

Reply:
Sorry, my bad. I missed them. I will update all the spots to cuda11.5. thank you

Reply:
other spots were updated. thank you